cfme/utils/appliance/__init__.py

```diff
@@ -1455,41 +1469,23 @@ def is_web_ui_running(self, unsure=False):
         else:
             return unsure
 
-    def _evm_service_command(self, command, log_callback, expected_exit_code=None):
-        """Runs given systemctl command against the ``evmserverd`` service
-        Args:
-            command: Command to run, e.g. "start"
-            expected_exit_code: If the exit codes don't match, ApplianceException is raised
-        """
-        log_callback("Running command '{}' against the evmserverd service".format(command))
-        with self.ssh_client as ssh:
-            result = ssh.run_command('systemctl {} evmserverd'.format(command))
-
-        if expected_exit_code is not None and result.rc != expected_exit_code:
-            msg = ('Failed to {} evmserverd on {}\nError: {}'
-                   .format(command, self.hostname, result.output))
-            log_callback(msg)
-            raise ApplianceException(msg)
-
-        return result.rc
-
     @logger_wrap("Status of EVM service: {}")
     def is_evm_service_running(self, log_callback=None):
```

Comment:
I'd also get rid of those methods as well in favor of service calls.

Reply:
done in latest commit